ctapipe/instrument/camera/tests/test_readout.py

```diff
@@ -145,5 +145,8 @@ def test_hashing():
 @pytest.mark.parametrize("camera_name", camera_names)
 def test_camera_from_name(camera_name):
     """ check we can construct all cameras from name"""
-    camera = CameraReadout.from_name(camera_name)
-    assert str(camera) == camera_name
+
+    # these two don't have readout definitions on the dataserver
+    if camera_name not in ["MAGICCam", "Whipple109"]:
```

Comment:
should we test one of those? I think we only test LSTCam (at least on my machine), which will then not really test the new behaviour introduced in this PR

Reply:
done